python/pynamodb/README.md

```diff
@@ -0,0 +1,49 @@
+### PynamoDB Example Script
+
+Based on the PynamoDB tutorial [here](https://pynamodb.readthedocs.io/en/latest/tutorial.html), with the boto configuration example based on [here](https://boto.readthedocs.io/en/latest/boto_config_tut.html). When it spins up, a simple python script that makes various PynamoDB calls runs. This also assumes that you have Docker running, because it uses the [DynamoDB Local Docker Image](https://hub.docker.com/r/amazon/dynamodb-local). Note that this sample app is for **testing/dev** purposes on a local instance of DynamoDB.
+
+For additional Python APM configuration options, refer to the API docs [here](http://pypi.datadoghq.com/trace/docs/integrations.html#pynamodb).
+
+#### Setup
+
+**Step 1)** Set up a `.env` file inside this directory (`./pynamodb`), so that the file is (`./pynamodb/.env`). Inside the file, add:
+
+```
+AWS_ACCESS_KEY_ID=<EXAMPLE KEY>
+AWS_SECRET_ACCESS_KEY=<EXAMPLE KEY>
+DD_API_KEY=<DATADOG API KEY>
+```
+
+Note: Since this is a local test database instance, you can leave it as `<EXAMPLE KEY>` and this will send traces to Datadog as long as the `<DATADOG API KEY>` is filled out.
```

Comment:
```suggestion
Note: Since this is a local test database instance, you can leave it as `<EXAMPLE KEY>` and this will send traces to Datadog as long as the `<DATADOG API KEY>` is provided.
```